public/app/features/alerting/unified/rule-list/FilterView.test.tsx

```diff
@@ -40,9 +40,6 @@ beforeEach(() => {
 const io = mockIntersectionObserver();
 
 describe('RuleList - FilterView', () => {
-  jest.setTimeout(60 * 1000);
-  jest.retryTimes(2);
```

Comment:
(I'm all for removing if unneeded, but just asking) Has something changed in the implementation that means these are more reliable now?

Reply:
Yes, we’ve changed how filters are applied to generators, which has significantly improved performance for the FilterView, making this test run much faster.